content/post/66-london-hack-week-report.md

```diff
@@ -0,0 +1,52 @@
+- **Friday** - Final review of the OKRs. As a prize for our very successful week, we all went to the British Museum to check out the Rosetta stone among many other cool things.
+
+In the end we ran Retrospective and asked "When is the next one! 😄"
+
+## Results from [SHOW ME WHAT YOU GOT 🗿](https://www.youtube.com/watch?v=LEUGPEVRDmU)
+
+<a href="https://www.youtube.com/watch?v=LEUGPEVRDmU"><img width="320" src="https://user-images.githubusercontent.com/1211152/47279522-ed3ed800-d5d1-11e8-85cc-5166f53808d6.png"></img></a>
+
+This is an IPFS Community favorite. A moment to report back all the great successes from the week! We do this report in lighting talk format. [Full playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ). Talks list:
+
+- [Bitswap/GraphSync Improvements and Module Architecture, Erik Ingenito](https://www.youtube.com/watch?v=6GKKwcG630M&index=8&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+- [Community Contributing Guidelines, David Dias](https://www.youtube.com/watch?v=rq2d-41nC44&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ&index=2&t=0s)
+- [Datastore Benchmarks, Eric Myhre](https://www.youtube.com/watch?v=3PlN0Xcjckk&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ&index=7)
+<!-- - [ipfs ls Wikipedia FAST, Hannah Howard](Waiting on GraphSync) -->
+- [IPLD Selectors & GraphSync, Juan Benet](https://www.youtube.com/watch?v=NhAQEKtPxHw&index=6&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+- [Offline First, Molly Mackinlay](https://www.youtube.com/watch?v=f1YUXacAuP0&index=9&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+- [2018 Q4 OKRs for go-ipfs, David Dias](https://www.youtube.com/watch?v=rf8xlRSVczE&index=5&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+- [Project Working Group OKRs, Molly Mackinlay](https://www.youtube.com/watch?v=cLUg_kBzl3I&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ&index=4)
+- [Provider Strategies, Michael Avila & Jan Winkelmann](https://youtu.be/Y_unBdXzRvU)
+- [Scalable PubSub with GossipSub, Dimitris Vyzovitis](https://www.youtube.com/watch?v=mlrf1058ENY&index=3&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+<!-- - [Simulations Framework for Providers Strategies, Jakub Sztandera](Waiting on Approval) -->
+- [The RFC RFC, Eric Myhre](https://www.youtube.com/watch?v=9m6VMnCTRhQ&index=2&list=PLuhRWgmPaHtRPl3Itt_YdHYA0g0Eup8hQ)
+
+## IPFS London Meetup - Show & Tell
+
+The fun would not be complete without an IPFS London Meetup with a selection of IPFS finest speakers and a surprise drive-by talk by Juan Benet himself, who hacked a thing while the meetup was happening. A post with the recorings will be published soon. Meanwhile, you can take a look at the pictures https://www.meetup.com/london-ipfs/photos/29361158/
+
+--------------------------------------
+
+Thank you to @858Heather, @jamiejn, @angiemaguire, @olizilla and @alanshaw for helping us putting this amazing hack week and IPFS London Meetup, y'all are a rockstars!!!!
```

Comment:
```suggestion
Thank you to @858Heather, @jamiejn, @angiemaguire, @olizilla and @alanshaw for helping us putting this amazing hack week and IPFS London Meetup!
```

`rockstars` should be avoided as an accolade. Histroically, folks who have been described as "rockstars" are also folks who over-work or otherwise don't consider the needs of there team. Any positive meaning it may have had got completely erased by it's use by tech recruiters in the
> "Rockstar developers wanted"

which priviledged those who had the free time to work around the clock.